privacyidea/static/templates/footer.html

```diff
@@ -28,7 +28,6 @@
 <script src="{{ instance }}/{{ "static/contrib/js/ngmodules/angular-idle.js" | versioned }}"></script>
 <script src="{{ instance }}/{{ "static/contrib/js/ngmodules/angular-gettext.js" | versioned }}"></script>
 <script src="{{ instance }}/{{  "static/components/translation/translations.js" | versioned }}"></script>
-<script src="{{ instance }}/{{ "static/contrib/js/ngmodules/highlight.js" | versioned }}"></script>
```

Comment:
Interesting, so was the hightlight.js a remnant from a previous try or did we actually used it?
Just for me to understand: And this is now done by our own `filter`, right?